.github/workflows/json.yaml

```diff
@@ -27,10 +27,7 @@ jobs:
         run: |
           git branch -D json || true
           git checkout -b json
-          mkdir -p docs
-          mkdir -p docs/scripts
-          mv *.json ./docs
-          find scripts -type f -name "*.js" -exec cp {} ./docs/scripts \;
```

Comment:
I would only replace this line and keep line 30-32.
`utils/copy_js_and_assets.py`